.github/release_template.md

```diff
@@ -0,0 +1,28 @@
+## ❗ Pre-merge checklist
+
+Please ensure these items are checked before merging.
+
+### 🔎 Smoke test
+
+- [ ] All CI checks pass
+- [ ] Docs and Storybook open in a browser
+- [ ] Successful integration test with GitHub Projects
```

Comment:
```suggestion
- [ ] Successful integration test with GitHub Projects as a primary consumer of PRC
```

Feel free to take this suggestion or leave it, just adds a bit of context for why this step is important.

Reply:
+1 to @lesliecdubs's suggestion, but I'd change `PRC` to `Primer React`. We try to avoid acronyms in our documentation.